Setup/UpgradeSchema.php

```diff
@@ -86,5 +86,54 @@ public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $con
 
             $installer->endSetup();
         }
+
+        if (version_compare($context->getVersion(), '1.0.3', '<')) {
```

Comment:
Is this method idempotent?

Reply:
@codymullins  The code inside `version_compare` will only run once, with the module's version being updated after the upgrade script finishes.  It isn't technically idempotent, as it makes an API call which can have different results over time.

Reply:
Understood. Is this the common practice with Magento?